### ISSUE TITLE
Consider redundancy when checking for focusable descendants of aria-hidden elements

### ISSUE THREAD

**Issue by fkohrt** [dequelabs/axe-core#2780]
__Expectation__

Axe should take into account if an image's `aria-hidden` `figcaption` is identical with the image's `alt` text. Only if not identical, [[aria-hidden="true"] elements contain focusable descendants](https://web.dev/aria-hidden-focus/) should become a potential issue.

__Actual__

A focusable `aria-hidden` `figcaption` is always an issue.

__Motivation__

[Pandoc](https://pandoc.org/) converts...

```md
![description](example.png)
```

...to...

```html
<figure>
<img src="example.png" alt="description" /><figcaption aria-hidden="true">description</figcaption>
</figure>
```

`description` is already available as `alt`, therefore the figcaption is being hidden from screen readers to avoid redundancy (see jgm/pandoc#6495 for reasoning).

**Comment by fkohrt** [comment]
Nevermind, I encountered the issue in an instance where `alt` and `figcaption` where, in fact, not redundant, but I didn't notice. What the original issue suggests seems to be already implemented.
